pages/find-locations/index.md

```diff
@@ -1,6 +1,11 @@
 ---
-title: Facility Locator
+title: Find VA Locations | Veterans Affairs
 layout: page-react.html
+description: Find a VA medical center, clinic, hospital, cemetery, or a VA benefits office near you. Search by city, state, ZIP code, or service. Get wait times and directions. 
+tags: 
+  - VA location by zip code 
+  - VA clinic
+  - VA near me
```

Comment:
How come `tags` instead of `keywords`? Also, I think it would make sense for this to be a plain string instead of a list, because they're all rendering under the same meta tag, rather than each in their own -

```
<meta name="keywords" content="HTML,CSS,XML,JavaScript">
```

Reply:
Ah I thought this was strictly a `vagov-content` repo fix. I used `tags` because I didn't see the where `keywords` were being set in the `/src/site/includes/metatags.liquid` on `vets-website`, but I saw tags [defined here](https://github.com/department-of-veterans-affairs/vets-website/blob/13e7ded516fa485852159e1bac9ce7d6f8bddd72/src/site/includes/metatags.liquid#L31)

I'll add `keywords` into the `metatags.liquid` template

Reply:
Oh, I'm not sure. Can you follow up in the ticket for clarity on exactly what meta property they're asking for? The word `Keywords` is used, so that's what I assumed.